src/main/java/net/sf/jabref/importer/fileformat/BibTeXMLHandler.java

```diff
@@ -32,17 +31,17 @@
  */
 class BibTeXMLHandler extends DefaultHandler {
 
+    private static final String BIBTEXML_URI = "http://bibtexml.sf.net/";
+
     private List<BibEntry> bibitems;
 
     private BibEntry b; // the entry being read
 
     // XML parsing stuff
     private String currentChars;
 
+
     public List<BibEntry> getItems() {
-        if (bibitems == null) {
-            return Collections.emptyList();
-        }
```

Comment:
Personal discussion: quick fix: keep the if check; initialization at class variable; reset at EndDocument. - Test case is obsolete, but you can add it to achieve 100% coverage.

A good redesign is a private static class or an anonym class.